doc/man7/EVP_KDF-KB.pod

```diff
@@ -21,23 +21,36 @@ The supported parameters are:
 
 =over 4
 
-=item B<OSSL_KDF_PARAM_PROPERTIES> ("properties") <UTF8 string>
+=item "properties" (B<OSSL_KDF_PARAM_PROPERTIES>) <UTF8 string>
 
-=item B<OSSL_KDF_PARAM_DIGEST> ("digest") <UTF8 string>
+=item "mode" (B<OSSL_KDF_PARAM_MODE>) <UTF8 string>
 
-=item B<OSSL_KDF_PARAM_DIGEST> ("mac") <UTF8 string>
+=item "mac" (B<OSSL_KDF_PARAM_MAC>) <UTF8 string>
 
-=item B<OSSL_KDF_PARAM_KEY> ("key") <octet string>
+=item "digest" (B<OSSL_KDF_PARAM_DIGEST>) <UTF8 string>
 
-=item B<OSSL_KDF_PARAM_SALT> ("salt") <octet string>
+=item "cipher" (B<OSSL_KDF_PARAM_DIGEST>) <UTF8 string>
 
-=item B<OSSL_KDF_PARAM_INFO> ("info") <octet string>
+=item "key" (B<OSSL_KDF_PARAM_KEY>) <octet string>
+
+=item "salt" (B<OSSL_KDF_PARAM_SALT>) <octet string>
+
+=item "info (B<OSSL_KDF_PARAM_INFO>) <octet string>
+
+=item "seed" (B<OSSL_KDF_PARAM_SEED>) <octet string>
 
 =back
 
-The parameters key, salt, and info correspond to KI, Label, and Context
-(respectively) in SP800-108.  As in that document, salt and info are optional
-and may be omitted.  Currently, only HMAC is supported for mac.
+The mode paramater determines which flavor of KBKDF to use - currently the
```

Comment:
NIT: parameter

Reply:
Early morning mind goes "para-matter"...  beyond normal parameters?  Strangely, it seems fitting.

I think I need more coffee...